app/electron/main.ts

```diff
@@ -311,18 +311,19 @@ function startElecron() {
       const octokit = new Octokit();
 
       async function fetchRelease() {
-        const githubReleaseURL = `GET /repos/{owner}/{repo}/releases/latest`;
+        const githubReleaseURL = `GET /repos/{owner}/{repo}/releases`;
+        // get me all the releases -> default decreasing order of releases
```

Comment:
These changes should also be in the original commit that introduces the logic.

If you want to highlight them, you can use the commit body.